app/javascript/components/concept-map/Concept.tsx

```diff
@@ -10,26 +12,31 @@ import {
   Visibility,
 } from './helpers/concept-visibility-handler'
 import { wrapAnimationFrame } from './helpers/animation-helpers'
-import { PureExerciseProgressBar } from './ExerciseProgressBar'
 
 export const Concept = ({
   slug,
   name,
   web_url,
+  tooltip_url,
```

Comment:
We're using camelCase for the other properties. Can we use them here too?

Reply:
I think these are like this because these properties are coming from the rails api. So either I need to use camelcase in rails, or snake case in typescript.

Reply:
@kntsoriano has a helper for this!

Reply:
help me @kntsoriano, you're my only hope!

Reply:
@neenjaw I use this to import the helper: `import { camelizeKeys } from 'humps'`

We usually apply the helper when we get the response from the API. But in your case, there is no API request, so I would probably camelize the keys inside `application.tsx`. Hope that helps!

Reply:
Yes! Thanks!

Reply:
(This).

Reply:
done

Reply:
Thanks for the tips!